client/src/components/NavBarFooter.js

```diff
@@ -53,6 +53,10 @@ export default function NavBarFooter({ journeyId }) {
     return `/journeys/${id}`;
   };
 
+  const statisticslink = function(id) {
+    return `/statistics/${id}`;
+  };
+
```

Comment:
add `props`to function and instead
```suggestion
const journeyId = props.match.params.journeyId
```